doctr/file_utils.py

```diff
@@ -35,6 +35,20 @@
     logging.info("Disabling PyTorch because USE_TF is set")
     _torch_available = False
 
+# Compatibility fix to make sure tensorflow.keras stays at Keras 2
+if "TF_USE_LEGACY_KERAS" not in os.environ:
+    os.environ["TF_USE_LEGACY_KERAS"] = "1"
+
+elif os.environ["TF_USE_LEGACY_KERAS"] != "1":
+    raise ValueError(
+        "docTR is only compatible with Keras 2, but you have explicitly set `TF_USE_LEGACY_KERAS` to `0`. "
+    )
+
+
+def ensure_keras_v2() -> None:  # pragma: no cover
```

Comment:
The breaking point here is that doctr needs to be imported before tensorflow:

two options:
- we keep the ensure_keras_v2 to make it explicit
- we import doctr (in the training / eval scripts) first and ignore ruff's (isort)

What would you prefer ?

Reply:
Keep `ensure_keras_v2` function, because option 2 will be a hell to maintain in my opinion

Reply:
Yes i agree :D 
@odulcy-mindee btw could you check your slack ? ^^